ui/src/app/utils/user-apps-utils.spec.tsx

```diff
@@ -127,33 +127,36 @@ describe('User Apps Helper functions', () => {
     expect(sidebarActiveIconStore.value).toBeNull();
 
     userAppsUtils.openAppOrConfigPanel(
-      'ws',
-      'wsid',
+      'ns',
+      'terraName',
       [{ status: AppStatus.STARTING, appType: AppType.RSTUDIO }],
       UIAppType.RSTUDIO,
       navigate
     );
 
     // Since RStudio is NOT in running state this will open the RStudio config side panel and
     // There will be no navigation to any other page
-    expect(mockNavigate).not.toBeCalled();
+    expect(mockNavigate).not.toHaveBeenCalled();
```

Comment:
unrelated: previous call was deprecated.  This was our only instance in the whole repo.